### ISSUE TITLE
Cannot GET /energiefluss/index.html

### ISSUE THREAD

**Issue by desq42** [SKB-CGN/ioBroker.energiefluss#161]
Hi.

"Cannot GET /energiefluss/index.html" ist das einzige was ich angezeigt bekomme.

Allem Anschein nach mache ich was falsch. Ist meine Web-Instanz falsch konfiguriert?
Ich bin für jeden Tipp dankbar.

![Bildschirmfoto 2023-03-17 um 10 49 54](https://user-images.githubusercontent.com/18470786/225871040-570a92a9-2c41-4eba-a90c-3ba95527b521.png)

Danke.

**Comment by SKB-CGN** [comment]
Was passiert, wenn Du "index.html" am Ende weglässt?

**Comment by SKB-CGN** [comment]
Hast Du den Adapter installiert und eine Instanz angelegt? Diese wird bei Github Installationen nicht automatisch erzeugt.

Ist der Energiefluss in der linken Seitenlasche zu sehen?

**Comment by desq42** [comment]
Ja und ja… 
![Bildschirmfoto 2023-03-17 um 11 05 53](https://user-images.githubusercontent.com/18470786/225874691-3294cbd2-e7d6-4aae-8b61-60e4f73e1085.png)

**Comment by SKB-CGN** [comment]
Ok, was erscheint, wenn Du auch /energiefluss am Ende weg lässt.

Also nur http://<ip> : 8082

**Comment by desq42** [comment]
"Cannot GET /"
Ich denke es liegt nicht an dem Energiefluss-Adapter, sondern irgendwas stimmt mit meinem Web.0 Adapter nicht…

**Comment by SKB-CGN** [comment]
Genau, das wollte ich heraus finden. Hak' mal das "Simple-Api" an und schau nochmal.

**Comment by desq42** [comment]
Du meinst in der Web.0 Instanz? Habe ich gemacht. Keine Änderung.

**Comment by SKB-CGN** [comment]
Ist auf dem Ding eine Firewall, die etwas blockt?

Kommen im Protokoll irgendwelche "Verbindungen" zum Web Adapter?

**Comment by desq42** [comment]
Nope. Ich habe was gefunden. Liegt an meinem Lametric-Adapter.
Sofern ich den stoppe geht's. Der braucht wohl seine eigene Instanz.
